### PR TITLE
[Ready]Makes space wind great again!

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Space wind is now a scaling percent to move on pressure differences.
Space wind percent to move scales with pressure amount, and other modifers:
Actively moving (as in holding down a move key) reduces chance of getting moved by space wind by 30%
Having a wall or dense anchored object to your left or right (in terms of direction to be moved) will lower your chance to be moved in space wind by 20% in each direction. (so a wall on both sides is 40%) (Only applies to mob's with limbs)

:cl:
tweak: Space wind (movement from pressure differentials (like a hole to space)) has been improved.
tweak: Space wind's rate of movement now scales with pressure differential amount.
experiment: Actively moving (as in holding down a move key) reduces chance/rate of getting moved by space wind.
add: Having a wall or dense anchored object to your left or right will lower the chance/rate of getting moved by space wind for each direction (IE: a wall on both sides of you will be a double reduction)
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -645,6 +645,33 @@ Sorry Giacom. Please don't be mad :(
...
 		return pick("trails_1", "trails_2")
 
+/mob/living/experience_pressure_difference(pressure_difference, direction, pressure_resistance_prob_delta = 0)
+	if (client && client.move_delay >= world.time + world.tick_lag*2)
+		pressure_resistance_prob_delta -= 30
+
+	var/list/turfs_to_check = list()
+
+	if (has_limbs)
+		var/turf/T = get_step(src, angle2dir(dir2angle(direction)+90))
```
Comment (RemieRichards) [inline code/modules/mob/living/living.dm]: can't you reuse the proc that finds what you can grab onto in space?

Comment (MrStonedOne) [inline code/modules/mob/living/living.dm]: no, because that doesn't check anchored

**Comment by Fox-McCloud** [comment]
Why not just bring the old time-based cooldown back for space wind.

spacewind is just plain annoying; high fast depressurization is neato, but the currents are just plain annoying.

**Comment by MMMiracles** [comment]
I mean you kinda need the wind part if you're going to have fast depressurization, where else is the air going otherwise? Just vanishing?

**Comment by MrStonedOne** [comment]
fox, this makes them less annoying unless it's actually a lot of air. making it scale with pressure difference amount just makes sense.

Making it more annoying in a open area vs a 1 tile area where you have stuff to grab onto just makes sense.

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -348,6 +348,11 @@
 	..()
 	update_icon()
 
+/obj/item/weapon/paper/experience_pressure_difference(pressure_difference, direction)
+	if(last_high_pressure_movement_air_cycle < SSair.times_fired)
+		last_high_pressure_movement_air_cycle = SSair.times_fired
+		step(src, direction)
+
```
Comment (MrPerson) [inline code/modules/paperwork/paper.dm]: Why is this necessary?

Comment (MrStonedOne) [inline code/modules/paperwork/paper.dm]: oh, at first I was worried about divide by zero memes, but i added a check anyways, so i guess this could be removed.
